terraso_backend/apps/soil_id/models/depth_dependent_soil_data.py

```diff
@@ -13,13 +13,298 @@
+    PH_TESTING_SATURATED_PASTE_EXTRACT = "saturated paste extract"
+
+    PH_TESTING_SOLUTIONS = (
+        (PH_TESTING_SOIL_WATER_1_1, _("1:1 soil/water")),
+        (PH_TESTING_SOIL_WATER_1_2, _("1:2 soil/water")),
+        (PH_TESTING_SOIL_WATER_1_2_5, _("1:2.5 soil/water")),
+        (PH_TESTING_SOIL_WATER_1_5, _("1:5 soil/water")),
+        (PH_TESTING_SOIL_CACL2_1_1, _("1:1 soil/0.1 M CaCL2")),
+        (PH_TESTING_SOIL_CACL2_1_2, _("1:2 soil/0.1 M CaCL2")),
+        (PH_TESTING_SOIL_CACL2_1_5, _("1:5 soil/0.1 M CaCL2")),
+        (PH_TESTING_SOIL_KCL_1_1, _("1:1 soil/1.0 M KCL")),
+        (PH_TESTING_SOIL_KCL_1_2_5, _("1:2.5 soil/1.0 M KCL")),
+        (PH_TESTING_SOIL_KCL_1_5, _("1:5 soil/1.0 M KCL")),
+        (PH_TESTING_SATURATED_PASTE_EXTRACT, _("Saturated Paste Extract")),
+        (OTHER, _("Other")),
+    )
+    ph_testing_solution = models.CharField(null=True, choices=PH_TESTING_SOLUTIONS)
+
+    PH_TESTING_INDICATOR_STRIP = "pH indicator strip"
+    PH_TESTING_INDICATOR_SOLUTION = "pH indicator solution"
+    PH_TESTING_METER = "pH meter"
+
+    PH_TESTING_METHODS = (
+        (PH_TESTING_INDICATOR_STRIP, _("pH indicator strip")),
+        (PH_TESTING_INDICATOR_SOLUTION, _("pH indicator solution")),
+        (PH_TESTING_METER, _("pH meter")),
+        (OTHER, _("other")),
+    )
+    ph_testing_method = models.CharField(null=True, choices=PH_TESTING_METHODS)
+
+    soil_organic_carbon = models.DecimalField(
+        null=True,
+        max_digits=4,
+        decimal_places=1,
+        validators=[MinValueValidator(0), MaxValueValidator(100)],
+    )
+
+    soil_organic_matter = models.DecimalField(
+        null=True,
+        max_digits=4,
+        decimal_places=1,
+        validators=[MinValueValidator(0), MaxValueValidator(100)],
+    )
+
+    SOIL_TESTING_DRY_COMBUSTION = "dry combustion"
+    SOIL_TESTING_WET_OXIDATION = "wet oxidation (Walkey-Black)"
+    SOIL_TESTING_LOSS_ON_IGNITION = "loss-on-ignition"
+    SOIL_TESTING_REFLECTANCE_SPECTROSCOPY = "reflectance spectroscopy"
+    SOIL_TESTING_FIELD_REFLECTOMETER = "field reflectometer"
+
+    SOIL_TESTING_METHODS = (
+        (SOIL_TESTING_DRY_COMBUSTION, _("Dry combustion")),
+        (SOIL_TESTING_WET_OXIDATION, _("Wet oxidation (Walkey-Black)")),
+        (SOIL_TESTING_LOSS_ON_IGNITION, _("Loss-on-ignition")),
+        (SOIL_TESTING_REFLECTANCE_SPECTROSCOPY, _("Reflectance spectroscopy")),
+        (SOIL_TESTING_FIELD_REFLECTOMETER, _("Field reflectometer")),
+        (OTHER, _("Other")),
+    )
+    soil_organic_carbon_testing = models.CharField(null=True, choices=SOIL_TESTING_METHODS)
+    soil_organic_matter_testing = models.CharField(null=True, choices=SOIL_TESTING_METHODS)
+
+    sodium_absorption_ratio = models.DecimalField(
+        null=True, max_digits=100, decimal_places=2, validators=[MinValueValidator(0)]
+    )
+
+    bedrock = models.PositiveIntegerField(null=True)
+
+    NONEFFERVESCENT = "noneffervescent - No bubbles form"
```

Comment:
@ltseng Could you review that the carbonates data inputs are accurate? Thanks!

Reply:
Looks good to me!